recipes/s2geometry/build.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+set -eu
+
+mkdir build
+cd build
+
+### Create Makefiles
+cmake \
+      -DCMAKE_PREFIX_PATH=$PREFIX \
+      -DCMAKE_INSTALL_PREFIX=$PREFIX \
+      -DCMAKE_INSTALL_LIBDIR=lib \
+      -DCMAKE_BUILD_TYPE=Release \
+      -DBUILD_SHARED_LIBS=ON \
+      -DBUILD_EXAMPLES=OFF \
+      -UGTEST_ROOT \
+      $SRC_DIR
+
+### Build
+cmake --build . -- -j${CPU_COUNT}
+
```

Comment:
Is there a make check/test that we can run here?

Reply:
Yes, s2geometry provides many tests, but it's cmake config currently requires getting and building googletest from source. I'm happy to patch `CMakeLists.txt` to use instead the `gtest` library that is shipped with conda-forge, if needed. I tested it locally (MacOS) and it works well. Or I can add a couple of commands to clone the `gtest` git repository...

Reply:
If that is too much trouble we can leave it for later in the feedstock. Let me know when you are "done."